conans/client/cache/cache.py

```diff
@@ -58,7 +59,7 @@ def __init__(self, base_folder, store_folder, output):
         super(ClientCache, self).__init__(self._store_folder)
         self.editable_packages = EditablePackages(self.conan_folder)
 
-    def package_layout(self, ref, short_paths=False):
+    def package_layout(self, ref, short_paths=False, *args, **kwargs):
```

Comment:
Given the change I did here (https://github.com/conan-io/conan/pull/4496/commits/2a127febf8939e9dbfeab6f3e49b9513a261516a) to fix the test that was failing in Windows (it was not looking into the short_paths) I wonder if we should change this default value from `short_paths=False` to `short_paths=None`, I think it would be safer.

WDYT, @memsharded?

Reply:
Yes, probably, until we fix that and force a explicit definition of it.